web/src/main/java/com/bazaarvoice/emodb/web/scanner/rangescan/LocalRangeScanUploader.java

```diff
@@ -186,7 +186,7 @@ public void stop()
 
     @Override
     public RangeScanUploaderResult scanAndUpload(
-            final int taskId, ScanOptions options, final String placement, ScanRange scanRange, final TableSet tableSet, Date stashStartTime)
+            final int taskId, ScanOptions options, final String placement, ScanRange scanRange, final TableSet tableSet, Date compactionControlTime)
```

Comment:
Minor:  Can you also change the parameter name to `compactionControlTime` in the `RangeScanUploader` interface?